### DISCUSSION TITLE
Redirect websockets

### DISCUSSION THREAD

**Discussion by BadriAhmed** [fastapi/fastapi#8892] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Websocket Redirect
I have two services, one have a websocket server on it for live transcription , but the client can only connect with the first one. What I want to do is in the following image : 
![ws_redirect](https://user-images.githubusercontent.com/36994416/121342873-ffecb380-c919-11eb-97c4-5ea415b0db3f.png)

 I checked the documentation, and the closest I can get is from the [RedirectResponse section ](https://fastapi.tiangolo.com/advanced/custom-response/?h=redirect#redirectresponse) : 

```Python
from fastapi import FastAPI
from fastapi.responses import RedirectResponse

app = FastAPI()


@app.get("/typer")
async def read_typer():
    return RedirectResponse("https://typer.tiangolo.com")
```

I would apperciate any help.
Thank you !

**Comment by noodave** [comment]
any updates? ran into this need as well..

**Comment by BadriAhmed** [comment]
Unfortunately not @noodave.
You can find the StackOverflow question here too:
https://stackoverflow.com/questions/67902923/websocket-redirect-using-fastapi

I ended up using a bridge for that solution, apparently, websocket redirection is an infrequent use case

**Comment by YuriiMotov** [comment, marked as answer]
Redirect would mean that client tries to connect to the first endpoint but gets redirect response and then connects to second endpoint. That means that second endpoint should also be available for connection.

If this is not what you need, you can implement the following logic: use [websocket client library](https://github.com/websocket-client/websocket-client) to send all data received from user (via first websocket endpoint) to second endpoint, and then send all data received from second endpoint back to user via first endpoint.

A bit lazy to write code, but if somebody is interested I will give a code example.
